lib/api/statistics.py

```diff
@@ -0,0 +1,82 @@
+from typing import Optional, Union
+from datetime import datetime, timedelta
+from rest_framework import viewsets, status
```

Comment:
Styleguide suggests separating the import groups with one empty line: built-in modules, third-party-modules
```suggestion

from rest_framework import viewsets, status
```